wa/starter/src/test/java/org/apache/syncope/wa/pac4j/saml/BaseSyncopeWASAML2Client.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Can you move this package under `org.apache.syncope.wa.starter` which should be the root package of this module?